protos/sources/vega/governance.proto

```diff
@@ -120,6 +118,10 @@ message NewMarketConfiguration {
   }
   // Decimal places for order sizes, sets what size the smallest order / position on the futures market can be.
   int64 position_decimal_places = 6;
+  // DEPRECATED: Use LiquiditySLAParameters instead.
```

Comment:
```suggestion
  // DEPRECATED: Use liquidity SLA parameters instead.
```
I think these are displayed differently in REST / gRPC, which is why I've reformatted it